google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/CopyBackupRequest.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.admin.v2.models;
+
+import com.google.api.core.InternalApi;
+import com.google.cloud.bigtable.admin.v2.internal.NameUtil;
+import com.google.common.base.Objects;
+import com.google.common.base.Preconditions;
+import com.google.protobuf.util.Timestamps;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+import org.threeten.bp.Instant;
+
+/** Fluent wrapper for {@link com.google.bigtable.admin.v2.CopyBackupRequest} */
+public final class CopyBackupRequest {
+  private final com.google.bigtable.admin.v2.CopyBackupRequest.Builder requestBuilder =
+      com.google.bigtable.admin.v2.CopyBackupRequest.newBuilder();
+  private final String sourceBackupId;
+  private final String sourceClusterId;
+  private final String sourceInstanceId;
+  private final String sourceProjectId;
+
+  private String destClusterId;
+
+  /**
+   * Create a {@link CopyBackupRequest} object. It assumes the source backup is located in the same
+   * instance as the destination backup. To copy from a backup in another instance, use {@link
+   * #of(String, String, String) of} method. To copy from a backup in another instance, use {@link
+   * #of(String, String, String, String) of} method
+   */
+  public static CopyBackupRequest of(String clusterId, String backupId) {
+    CopyBackupRequest request = new CopyBackupRequest(null, null, clusterId, backupId);
```

Comment:
This doesn't seem to match up - `clusterId` is the 3rd parameter, but the 3rd parameter should be `instanceId` per the method signature: https://github.com/googleapis/java-bigtable/pull/1398/files#diff-f22bea44852cc477c88df0651b15890d220ccbfdf7f998d0e97bc6c30d409b8bR65

Reply:
The private method is defined as: 
private CopyBackupRequest(
      @Nullable String sourceProjectId,
      @Nullable String sourceInstanceId,
      @Nonnull String sourceClusterId,
      @Nonnull String sourceBackupId);
The 3rd parameter is ClusterId, the order of parameters of the private method and the public ones are different, should I match them?

Reply:
got it - in this case, the private method should follow the order of the public ones to avoid confusion (like I just had :)). It will also change the order so the `null` parameters will go toward the end, which is more typical and expected.

Reply:
sg, I'll udpate!